### ISSUE TITLE
AB Conversion - Debug scenes skipped assets

### ISSUE THREAD

**Issue by pravusjif** [decentraland/unity-renderer#424]
Every now and then we see some scenes exiting with error code 4, meaning that not all of its assets were converted, we should debug those cases and see what can be done to fix those issues.

All the relevant info and instructions is at the [ABs Playbook](https://playbooks.decentraland.systems/world-team/asset-bundles.html), and the scene conversion logs can be seen at https://unityab.decentraland.io/#/logentry

From what I could see (for example in [this case](https://unityab.decentraland.io/#/logentry/397541/show) with [this log](https://dev-assetbundles-8af4af4.s3.amazonaws.com/logs//opt/unity/Editor/export_log_QmPiUjxMsC2xnn87GY4nUubqeYj2spSCHex48RMwriKCJe_1620729512.txt)) it's normally a problem with some materials name

**Comment by olavra** [comment]
Is this still happening with the new asset bundles? @pravusjif

**Comment by pravusjif** [comment]
> Is this still happening with the new asset bundles? @pravusjif

Yes, this happens quite frequently. In some cases it's probably due to bad-generated GLTFs (for example pointing to unexistent textures)
